addons/medical_gui/functions/fnc_updateInjuryList.sqf

```diff
@@ -102,6 +139,11 @@ private _totalIvVolume = 0;
 if (_totalIvVolume >= 1) then {
     _entries pushBack [format [localize ELSTRING(medical_treatment,receivingIvVolume), floor _totalIvVolume], [1, 1, 1, 1]];
 };
+// Indicate if target has a pnuemothorax
+if (GET_PNEUMO(_target) && {_selectionN  == 1}) then {
+    _entries pushBack [localize LSTRING(STATUS_HAS_PNUEMOTHORAX), [1, 0, 0, 1]];
+};
```

Comment:
We definitely need a way to diagnose this without just seeing this, stethoscope would be the most familiar for people, but maybe we could also tie into breath sounds? Play a different sound that indicates they are having troubles breathing, maybe coughing too?

Reply:
The sound thing is nice, but normally it is not that easy to say it from the sounds without a stethoscope, but maybe a "Closer look at the chest" would diagnose it, since half of the body is not moving while ex or inhale